{
  "info": {
    "author": "Daniel Perrefort",
    "author_email": "",
    "bugtrack_url": null,
    "classifiers": [
      "Natural Language :: English",
      "Operating System :: OS Independent",
      "Programming Language :: Python",
      "Programming Language :: Python :: 3",
      "Programming Language :: Python :: 3.6",
      "Programming Language :: Python :: 3.7",
      "Programming Language :: Python :: 3.8",
      "Programming Language :: Python :: 3.9"
    ],
    "description": "",
    "description_content_type": "",
    "docs_url": null,
    "download_url": "",
    "downloads": {
      "last_day": -1,
      "last_month": -1,
      "last_week": -1
    },
    "home_page": "",
    "keywords": "Django HTTP Error Pages",
    "license": "GPL v3",
    "maintainer": "",
    "maintainer_email": "",
    "name": "django-easy-error",
    "package_url": "https://pypi.org/project/django-easy-error/",
    "platform": null,
    "project_url": "https://pypi.org/project/django-easy-error/",
    "project_urls": null,
    "release_url": "https://pypi.org/project/django-easy-error/1.0.2/",
    "requires_dist": [
      "django"
    ],
    "requires_python": ">=3.6",
    "summary": "Easy template integration for django error pages",
    "version": "1.0.2",
    "yanked": false,
    "yanked_reason": null
  },
  "last_serial": 14062760,
  "urls": [
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "78d28c37beb66d740c0e50ee0a4bca80b72e06a28d97aca26320dae32edadec7",
        "md5": "3fe010918745732fc2840fdfec54d98a",
        "sha256": "885f2f8f2101604e05ce550a995a7d929c6c72806bb1a63f0d25b67190f3a64a"
      },
      "downloads": -1,
      "filename": "django_easy_error-1.0.2-py3-none-any.whl",
      "has_sig": false,
      "md5_digest": "3fe010918745732fc2840fdfec54d98a",
      "packagetype": "bdist_wheel",
      "python_version": "py3",
      "requires_python": ">=3.6",
      "size": 17014,
      "upload_time": "2022-06-07T22:40:18",
      "upload_time_iso_8601": "2022-06-07T22:40:18.498170Z",
      "url": "https://files.pythonhosted.org/packages/78/d2/8c37beb66d740c0e50ee0a4bca80b72e06a28d97aca26320dae32edadec7/django_easy_error-1.0.2-py3-none-any.whl",
      "yanked": false,
      "yanked_reason": null
    },
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "7bbcdd51417a313bca5b60506093f446b8d384756d9d9e0a3d13945b626847ad",
        "md5": "9df093ea2438de0dd9ac264f81d4638d",
        "sha256": "c2780b398c31e702c9f708059d8d223e446efed1d4f41860ab5d22de799d7e37"
      },
      "downloads": -1,
      "filename": "django-easy-error-1.0.2.tar.gz",
      "has_sig": false,
      "md5_digest": "9df093ea2438de0dd9ac264f81d4638d",
      "packagetype": "sdist",
      "python_version": "source",
      "requires_python": ">=3.6",
      "size": 16599,
      "upload_time": "2022-06-07T22:40:19",
      "upload_time_iso_8601": "2022-06-07T22:40:19.645137Z",
      "url": "https://files.pythonhosted.org/packages/7b/bc/dd51417a313bca5b60506093f446b8d384756d9d9e0a3d13945b626847ad/django-easy-error-1.0.2.tar.gz",
      "yanked": false,
      "yanked_reason": null
    }
  ],
  "vulnerabilities": []
}